src/System.Runtime.Numerics/tests/BigInteger/cast_from.cs

```diff
@@ -686,7 +703,32 @@ public static void RunDecimalExplicitCastFromBigIntegerTests()
                 Assert.Throws<OverflowException>(() => VerifyDecimalExplicitCastFromBigInteger(value, bigInteger));
             }
         }
-        
+
+        /// <summary>
+        /// Test cast to Double on Very Large BigInteger more than (1 &lt&lt Int.MaxValue)
+        /// Tested BigInteger are: +/-pow(2, startShift + smallLoopShift * [1..smallLoopLimit] + Int32.MaxValue * [1..bigLoopLimit])
+        /// Expected double is positive and negative infinity
+        /// Note: 
+        /// ToString() can not operate such large values
+        /// </summary>
+        private static void DoubleExplicitCastFromLargetBigIntegerTests(int startShift, int bigShiftLoopLimit, int smallShift = 0, int smallShiftLoopLimit = 1)
```

Comment:
Small nit: I think there's an extra 't' in the method name ("Larget" -> "Large")